### PR TITLE
Adding expiration to jwt so it does not refresh every browser restart

### PR DESCRIPTION
**Pull request by dbrezack**
#### What's this PR do?
Fixes a possible issue with the browser not saving jwt tokens 
#### Related JIRA tickets:
https://jira.amida-tech.com/browse/RR-745
#### How should this be manually tested?
See this PR.
https://github.com/amida-tech/recruitment-registry-client/pull/185
#### Any background context you want to provide?
JWT tokens without an expiration time / age will not be saved in browser. 
#### Screenshots (if appropriate):

### CODE REVIEW

**Comment by dbrezack** [comment]
@au2 configuration option has been added to the config and the Readme. See: RECREG_TOKEN_EXPIRATION_MINTUES

File: README.md [hunk 1]
```diff
@@ -72,7 +72,9 @@ RECREG_CORS_ORIGIN=http://localhost:4000
 RECREG_ZIP_BASE_URL="http://www.zipcodeapi.com/rest/"
 RECREG_ZIP_API_KEY=xxx
 RECREG_ZIP_DISTANCE=50
-RECREG_ZIP_UNITS=mile
+RECREG_ZIP_UNITS=mile
+RECREG_TOKEN_EXPIRATION_MINTUES=43200
+RECREG_ZIP_UNITS=mile
```
Comment (unprofessional) [inline README.md]: Duplicate line, please remove

**Comment by unprofessional** [comment]
Functionality verified. Just need to remove the duplicate README line referenced above.

**Comment by dbrezack** [comment]
@unprofessional README.md Fixed

**Pull request merged**